.editorconfig

```diff
@@ -61,6 +61,7 @@ csharp_using_directive_placement = outside_namespace:silent
 csharp_prefer_simple_using_statement = true:suggestion
 csharp_prefer_braces = true:silent
 csharp_style_namespace_declarations = block_scoped:silent
+csharp_style_namespace_declarations = file_scoped:warning
```

Comment:
I see the advantages of going on that approach @joelfoliveira and @brmagadutra, but IMO, I see no problem of keeping as a warning for two reasons:

- Since it is possible on the IDE to solve the warning for the entire solution, therefore, it is a straightforward refactor;
- As [this reference](https://github.com/dotnet/aspire/blob/main/.editorconfig#L286), the majority of the IDE rules are stated as warnings;

What do you think?

Reply:
The issue is that sometimes warnings get overlooked. 
But yeah, I don't have a strong opinion on this either because I also don't want these rules to be too intrusive.

Reply:
I understand, isn't a blocker, but I would prefer if this was an error to be easier to see, like Joel mentioned. @gsferreira what do you think?

Reply:
In this case, I would also classify it as an error.